components/sup/src/manager/service/mod.rs

```diff
@@ -292,36 +288,6 @@ impl Service {
         self.supervisor.check_process()
     }
 
-    pub fn register_metrics(&self) {
```

Comment:
Yeah, the "metrics" which were registered here were completely useless. It was a fake Gauge for "version number" of the software. We already have this in output from the http-gateway without this metric